src/event_logger.cpp

```diff
@@ -0,0 +1,101 @@
+#include "event_logger.hpp"
+
+#include "logger.hpp"
+
+#include <sdbusplus/asio/property.hpp>
+#include <sdbusplus/server.hpp>
+
+namespace vpd
+{
+std::unordered_map<types::SeverityType, std::string>
+    EventLogger::m_severityMap = {
+        {types::SeverityType::Notice,
+         "xyz.openbmc_project.Logging.Entry.Level.Notice"},
+        {types::SeverityType::Informational,
+         "xyz.openbmc_project.Logging.Entry.Level.Informational"},
+        {types::SeverityType::Debug,
+         "xyz.openbmc_project.Logging.Entry.Level.Debug"},
+        {types::SeverityType::Warning,
+         "xyz.openbmc_project.Logging.Entry.Level.Warning"},
+        {types::SeverityType::Critical,
+         "xyz.openbmc_project.Logging.Entry.Level.Critical"},
+        {types::SeverityType::Emergency,
+         "xyz.openbmc_project.Logging.Entry.Level.Emergency"},
+        {types::SeverityType::Alert,
+         "xyz.openbmc_project.Logging.Entry.Level.Alert"},
+        {types::SeverityType::Error,
+         "xyz.openbmc_project.Logging.Entry.Level.Error"}};
+
+std::unordered_map<types::PelMessage, std::string> EventLogger::m_message = {
```

Comment:
Same here. Can this also be marked as static?

Reply:
m_message is declared as static member.